src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -18,12 +18,26 @@
   <ItemGroup>
     <ReferenceFromRuntime Include="System.Private.CoreLib" />
     <ReferenceFromRuntime Include="CoreFx.Private.TestUtilities" />
-    <ReferenceFromRuntime Include="xunit.core" />
-    <ReferenceFromRuntime Include="Xunit.NetCore.Extensions" />
-    <ReferenceFromRuntime Include="xunit.assert" />
+
+    <!-- Keep in sync with test.targets -->
```

Comment:
This needs to be conditioned to the proper targetgroup -- if you try to build for UAP you would get an error because we're not using the same runners.

Reply:
I mean some of the references will not be present in UAP .